python/ccf/ledger.py

```diff
@@ -149,6 +164,200 @@ def _byte_read_safe(file, num_of_bytes):
     return ret
 
 
+class NodeStatus(IntEnum):
+    """These are the corresponding status meanings from the ccf.nodes table"""
+
```

Comment:
Although not covered by pep8, since they came later, Enums tend to be overwhelmingly all caps in Python (https://docs.python.org/3/library/enum.html#creating-an-enum, https://mail.python.org/pipermail/python-ideas/2016-September/042340.html), and they are consistently all caps in CCF in both Python and C++, so I would prefer if we could make this one the same.